apps/condo/domains/user/components/UserOrganizationsList.tsx

```diff
@@ -82,13 +83,21 @@ const OrganizationEmployeeItem: React.FC<IOrganizationEmployeeItem> = (props) =>
                         )}
                     />
                 </Col>
+                <Col lg={5} xs={10}>
+                    <Typography.Text type='secondary'>
+                        {InnMessage}
+                    </Typography.Text>
+                </Col>
+                <Col lg={18} xs={12} offset={1}>
+                    <NotDefinedField value={get(employee, ['organization', 'tin'])} />
```

Comment:
Why its `NotDefinedField`? I thought every organization must have one...

Reply:
https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/organization/schema/Organization.js#L108

Reply:
NotDeinedField is used everywhere in this component. I'd have left it like this tbh, maybe we can even rename it `OrganizationInfoField` (it is in org domain)